LR_Tourist/TouristWebApp/Controllers/HotelController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BLL.Model;
+using BLL.Services;
+using DA;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TouristWebApp.Controllers
+{
+    public class HotelController : Controller
+    {
+        private readonly HotelManagementService _hotelManagementService;
+        public HotelController(HotelManagementService hotelManagementService)
+        {
+            _hotelManagementService = hotelManagementService;
```

Comment:
Why logger absent here?
I guess would be better to log successful or failed operation with different LogLevel.

Reply:
fixed